app/components/input.hbs

```diff
@@ -0,0 +1,11 @@
+<label class={{@input-label-class}}>
+  {{@label}}
+  <input
+    class={{@input-class}}
+    type={{@type}}
+    placeholder={{@placeholder}}
+    value={{@value}}
+    onInput={{this.onChange}}
```

Comment:
Use `pick` from `ember-composable-helpers` and call the action directly.
https://github.com/DockYard/ember-composable-helpers#pick